src/nostr-profile/nostr-profile.ts

```
@@ -0,0 +1,819 @@
+      <div style="width: 100%; height: 100%; border-radius: 50%" class="skeleton"></div>
+      </div>
+      </div>
+    </div>
+    </div>
+    </div>
+    
+    // Sanitize and apply the HTML content
+    this.shadowRoot!.innerHTML = sanitizeHtml(loadingHTML);
+    <div class="profile_actions">
+    ${
+    this.ndkUser && this.ndkUser.npub && showFollow
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)